instrumentation/log/filters.go

```diff
@@ -39,6 +39,10 @@ func IgnoreErrorsMatching(pattern string) Filter {
 	return &errorRegexp{pattern: pattern}
 }
 
+func Nothing() Filter {
```

Comment:
I would call it `DiscardAll` because if you _filter nothing_, you're not supposed to filter at all.